argilla/src/argilla/_api/_datasets.py

```diff
@@ -48,13 +48,13 @@ def create(self, dataset: "DatasetModel") -> "DatasetModel":
 
     @api_error_handler
     def update(self, dataset: "DatasetModel") -> "DatasetModel":
-        json_body = dataset.model_dump()
+        json_body = dataset.model_dump(exclude_unset=True)
         dataset_id = json_body["id"]  # type: ignore
         response = self.http_client.patch(f"{self.url_stub}/{dataset_id}", json=json_body)
         response.raise_for_status()
         response_json = response.json()
         dataset = self._model_from_json(response_json=response_json)
-        self._log_message(message=f"Updated dataset {dataset.url}")
+        self._log_message(message=f"Updated dataset {dataset.id}")
```

Comment:
i would say the url is more useful for humans. why did you change this?

Reply:
@frascuchon, ahh okay. I did not realize this was not included. I created this issues to see if anyone from the community might be interested in pciking it up. https://github.com/argilla-io/argilla/issues/5195